test/test.js

```diff
@@ -77,6 +79,16 @@ describe('Argon2', () => {
       const hash = await argon2.hash(password, { associatedData, salt })
       assert.equal(hashes.withAd, hash)
     })
+
+    it('with associated data', async () => {
+      const hash = await argon2.hash(password, { associatedData, salt })
+      assert.equal(hashes.withAd, hash)
+    })
+
```

Comment:
Great resolution algorith, git!

This duped test needs to be removed...

Reply:
> I guess using the kats would be better but I have no clue how that file works.

Working on it ATM

> Great resolution algorith, git!
> 
> This duped test needs to be removed...

Haha 😄 Will do !